pkg/cloudprovider/aws/factory.go

```diff
@@ -16,18 +16,19 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
```

Comment:
Why..?

Reply:
I wanted to separate 3rd party imports vs project imports. I'm fine with having it either way. I can change it back if you'd like.

Reply:
We should be relying on a code formatter to make these decisions. Let's make sure we're not doing it manually, and that were using the same one.